README.md

```diff
@@ -9,3 +9,664 @@ ExpressRouterAdapter is available as a package on NPM for use with a module bund
+  ];
+  getRoutes(): IHTTPRoute[] {
+    return [
+      new RouterMetaBuilder()
+        .path('/pets/:petId')
+        .mediaType(this.oldPetMediaType)
+        .mediaType(this.petMediaType)
+        .allowAnonymous()
+        .get(({ petId }) => {
+          return this.pets[petId];
+        })
+    ];
+  }
+}
+
+```
+so now...
+```sh
+curl -H accept:application/json  http://localhost:4000/pets/1
+{"name":"poseidon hoguet"}
+curl -H accept:application/pet+json  http://localhost:4000/pets/1
+{"firstName":"poseidon","lastName":"hoguet"}
+```
+We also can re-use our formatter for our aggregates...
+
+```ts
+import { IHTTPRoute, RouterMetaBuilder } from '@symbiotic/express-router-adapter';
+import { autoinject } from 'aurelia-dependency-injection';
+
+interface IPet {
+  firstName: string;
+  lastName: string;
+  internalS3PathToPicture: string;
+}
+
+class OldPetMediaType {
+  mediaType: string = 'application/json';
+  formatForResponse({ firstName, lastName }: IPet): any {
+    return { name: `${firstName} ${lastName}` };
+  }
+}
+
+// tslint:disable-next-line: max-classes-per-file
+class PetMediaType {
+  mediaType: string = 'application/pet+json';
+  formatForResponse({ firstName, lastName }: IPet): any {
+    return { firstName, lastName };
+  }
+}
+
+// tslint:disable-next-line: max-classes-per-file
+@autoinject
+class PetsMediaType {
+  constructor(private petMediaType: PetMediaType) {}
+  mediaType: string = 'application/pets+json';
+  formatForResponse(pets: IPet[]): any {
+    return pets.map(this.petMediaType.formatForResponse);
+  }
+}
+
+// tslint:disable-next-line: max-classes-per-file
+@autoinject
+export class PetRouter {
+  constructor(private petMediaType: PetMediaType, private oldPetMediaType: OldPetMediaType, private petsMediaType: PetsMediaType){}
+  pets: IPet[] = [
+    { firstName: 'honey', lastName: 'hoguet', internalS3PathToPicture: '...' },
+    { firstName: 'poseidon', lastName: 'hoguet', internalS3PathToPicture: '...' }
+  ];
+  getRoutes(): IHTTPRoute[] {
+    return [
+      new RouterMetaBuilder()
+        .path('/pets/')
+        .mediaType(this.petsMediaType)
+        .allowAnonymous()
+        .get(() => {
+          return this.pets;
+        }),
+      new RouterMetaBuilder()
+        .path('/pets/:petId')
+        .mediaType(this.oldPetMediaType)
+        .mediaType(this.petMediaType)
+        .allowAnonymous()
+        .get(({ petId }) => {
+          return this.pets[petId];
+        })
+    ];
+  }
+}
+```
+
+Once you add a custom media type, only the media types you specify will work (you've opted out of the default application/json). This is a DX issue for consumers of your API and we include a helpful response to help navigate it.
+
+```sh
+curl -H 'accept:application/json'  http://localhost:4000/pets/
+{"status":406,"message":"The requested Accept format cannot be satisfied. Try one of the supported media types: application/pets+json"}
```

Comment:
nice to have formater these json responses to multi lines so they are more readable

Reply:
Also it will fix them being clipped

![image](https://user-images.githubusercontent.com/1964631/61544860-42e6ca80-aa14-11e9-819d-0c56dde153eb.png)